visual-js/visual-nightwatch/nightwatch/assertions/sauceVisualResults.js

```diff
@@ -58,6 +58,17 @@ exports.assertion = function sauceVisualResults(diffStatus, expected, msg) {
    * @param {function} callback
    */
   this.command = async function (callback) {
+    // Return only SKIPPED if in skip mode
+    if (isSkipMode()) {
+      const statusSummary = Object.fromEntries(
+        Object.values(DiffStatus).map((status) => [status, 0]),
+      );
+      statusSummary['SKIPPED'] = global.skipped;
+      return callback({
+        value: statusSummary[diffStatus],
+      });
```

Comment:
I don't get this code. In the end only `statusSummary[diffStatus]` is used. According to my understanding of the code this value is always `0` unless, `diffStatus == 'SKIPPED'`.

What is it supposed to do?

Reply:
Looks like it's supposed to emulate a `getVisualResults` check so they can assert against it (ex: assert that we have 1 unapproved snapshot after check - `sauceVisualResults(DiffStatus.Unapproved, 1)`). IMO this command should probably be deprecated / removed since it's bad practice while the diffs are async. Though, to keep the functionality as-is while adding skipped behavior it should probably be below to match the nightwatch builder API

```
callback({
    value: statusSummary[diffStatus],
});
return this;
```